.github/workflows/iroha2-dev-nightly.yml

```diff
@@ -15,7 +16,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Build wasm libs
-        run: ./scripts/build_wasm.sh libs
+        run: ./scripts/build_wasm.sh --target=libs
```

Comment:
why change to named argument instead of position based

Reply:
It was easier to validate when all arguments were named. I can revert if these must be position-based.

Reply:
revert